<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_coordonnees</class>
 <widget class="QDialog" name="Dialog_coordonnees">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fiche coordonnées</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../openjardin.qrc">:/images/adresses.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_idCoordonnees">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>COORDONNEES</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_societe"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="8" column="2">
        <widget class="QLineEdit" name="lineEdit_Pays"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Code postal</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2"/>
       </item>
       <item row="9" column="2">
        <widget class="QLineEdit" name="lineEdit_Fixe"/>
       </item>
       <item row="5" column="2">
        <widget class="QLineEdit" name="lineEdit_Adresse"/>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Fax</string>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Email</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLineEdit" name="lineEdit_Code_postal"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Ville</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="label_14">
         <property name="text">
          <string>Portable</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Prénom</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Pays</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Commentaires</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QTextEdit" name="textEdit_Commentaires"/>
       </item>
       <item row="11" column="2">
        <widget class="QLineEdit" name="lineEdit_Fax"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>Téléphone fixe</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QLineEdit" name="lineEdit_portable"/>
       </item>
       <item row="1" column="2">
        <widget class="QLineEdit" name="lineEdit_Societe"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Adresse</string>
         </property>
        </widget>
       </item>
       <item row="12" column="2">
        <widget class="QLineEdit" name="lineEdit_Email"/>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="lineEdit_Prenom"/>
       </item>
       <item row="3" column="2">
        <widget class="QLineEdit" name="lineEdit_Nom"/>
       </item>
       <item row="7" column="2">
        <widget class="QLineEdit" name="lineEdit_Ville"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Société</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_nouveau">
         <property name="text">
          <string>Nouvelle coordonnée</string>
         </property>
         <property name="icon">
          <iconset resource="../openjardin.qrc">
           <normaloff>:/images/add_48.png</normaloff>:/images/add_48.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Supprimer">
         <property name="text">
          <string>Supprimer</string>
         </property>
         <property name="icon">
          <iconset resource="../openjardin.qrc">
           <normaloff>:/images/remove_48.png</normaloff>:/images/remove_48.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Valider">
         <property name="text">
          <string>Valider</string>
         </property>
         <property name="icon">
          <iconset resource="../openjardin.qrc">
           <normaloff>:/images/Valider.png</normaloff>:/images/Valider.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Annuler">
         <property name="text">
          <string>Annuler</string>
         </property>
         <property name="icon">
          <iconset resource="../openjardin.qrc">
           <normaloff>:/images/icones/quitter.png</normaloff>:/images/icones/quitter.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../openjardin.qrc"/>
 </resources>
 <connections/>
</ui>
